product_docs/docs/epas/11/installing/linux_install_details/rpm_packages.mdx

```diff
@@ -169,36 +163,20 @@ apt list edb*
 
 ### Updating an RPM installation
```

Comment:
Do you think we can change this to "Updating a RPM installation"

Reply:
I changed wording to "Updating RPM installations"